### PR TITLE
italian translation of regression line article

### PR DESCRIPTION
**Pull request by stefanoanzani**
Hopefully I didn't mess up somewhere :D 
This is the translation for issue #19

### CODE REVIEW

File: content/posts/2021-01-26-ggplot-regression-line.it.Rmarkdown [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+```{r ggplot2_polynomial_regression_line}
+ggplot(mtcars, aes(qsec, hp)) +
+  geom_point() +
+  geom_smooth(method = "lm", formula = y ~ poly(x, 2))
+```
+
+<!-- Now it's your turn! Start a new R session, load some data, and create a ggplot with a linear regression line. Happy programming! -->
+
+Ora è il tuo turno! Apri una nuova sessione in R, carica dei dati e crea un ggplot con una retta di regressione lineare. Buona programmazione!
```
Comment (thomas-neitmann) [inline content/posts/2021-01-26-ggplot-regression-line.it.Rmarkdown:80]: Please add a sentence like "This post has been kindly translated from the original English by <your name or twitter handle with a link to your twitter/website/GitHub (whatever you like)>".

**Review by thomas-neitmann — CHANGES_REQUESTED** [review]
Thank you so much! Please have a look at my one comment.

File: content/posts/2021-01-26-ggplot-regression-line.it.Rmarkdown (hunk 1 above)
Comment (stefanoanzani) [reply content/posts/2021-01-26-ggplot-regression-line.it.Rmarkdown:80]: Ok i've added it and pushed the commit. Do I have to update the PR somehow?

Comment (thomas-neitmann) [reply content/posts/2021-01-26-ggplot-regression-line.it.Rmarkdown:80]: No, it's automatically added to the PR 🙂

**Review by thomas-neitmann — APPROVED** [review]
Thank you so much!! 🥳

**Pull request merged**